LeaderboardBackend.Test/Features/Users/SendConfirmationTests.cs

```diff
@@ -108,6 +109,13 @@ public async Task ResendConfirmation_EmailFailedToSend()
         client.DefaultRequestHeaders.Authorization = AuthenticationHeaderValue.Parse($"Bearer {token}");
         HttpResponseMessage res = await client.PostAsync(Routes.RESEND_CONFIRMATION, null);
         res.Should().HaveStatusCode(HttpStatusCode.InternalServerError);
+
+        ApplicationContext context = _scope.ServiceProvider.GetRequiredService<ApplicationContext>();
+        context.AccountConfirmations.Where(c =>
+            c.UserId == result.AsT0.Id &&
+            c.CreatedAt == Instant.FromUnixTimeSeconds(1) &&
+            c.ExpiresAt == Instant.FromUnixTimeSeconds(1) + Duration.FromHours(1)
+        ).Should().NotBeEmpty();
```

Comment:
Sorry for being picky but I just realized if the email failed to send then it shouldn't matter if the confirmation was created or not because the user can't use it.

Reply:
Yeah, it really doesn't matter to check for it